styles/new/base.css

```
@@ -41,3 +41,50 @@ ol {
     text-neutral-900
     dark:text-white;
 }
+
+table {
+  @apply w-full 
+  border-separate 
```

Please indent styles as shown here: [Example of a CSS Module](https://github.com/nodejs/nodejs.org/blob/main/COLLABORATOR_GUIDE.md#example-of-a-css-module)

```suggestion
    border-separate 
```